styles/style.css

```diff
@@ -396,20 +397,23 @@ button:hover {
   height: 200px;
   background-image: url("../img/icon-languages.svg");
   background-repeat: no-repeat;
+  margin-left: 35%;
 }
 
 .card_img2 {
   width: 370px;
   height: 200px;
-  background-image: url("../img/icon-languages.svg");
+  background-image: url("../img/frameworks-icon2.svg");
   background-repeat: no-repeat;
+  margin: auto;
 }
 
 .card_img3 {
   width: 370px;
   height: 200px;
-  background-image: url("../img/icon-languages.svg");
+  background-image: url("../img/skills-icon.svg");
   background-repeat: no-repeat;
+  margin: auto;
 }
 

```

Comment:
- Kindly use your media queries for the Desktop breakpoint in the CSS file. (the value for min-width property should be in range of 992px - 1100px). It's required in your project.

- Be DRY. your CSS file it's too long. If you added there your reset.css kindly create a new css files dedicated to this and reuse your code.

In your mobile view kindly fix this issue: 
![Screen Shot 2021-07-22 at 03 57 34](https://user-images.githubusercontent.com/22258274/126613924-5181cf37-0718-4164-b3aa-2898cfdfd8ac.png)